_ml-commons-plugin/algorithms.md

```diff
@@ -629,4 +629,79 @@ POST _plugins/_ml/_predict/logistic_regression/SsfQaoIBEoC4g4joZiyD
 
 ### Limitations
 
-Convergence metrics are not built into Tribuo's trainers. Therefore, ML Commons cannot indicate the convergence status through the ML Commons API.
\ No newline at end of file
+Convergence metrics are not built into Tribuo's trainers. Therefore, ML Commons cannot indicate the convergence status through the ML Commons API.
+
+## Metrics correlation
+
+The metrics correlation feature is an experimental feature released in OpenSearch 2.7. It can't be used in a production environment. To leave feedback on improving the feature, leave an issue in the [ML Commons repository](https://github.com/opensearch-project/ml-commons).
+{: .warning }
+
+The metrics correlation algorithm finds events in a set of metrics data. The algorithm defines events as a window in time where multiple metrics simultaneously display anomalous behavior. When given a set of metrics, the algorithm counts the number of events that occurred, when each event occurred, and determines which metrics were involved in each event.
+
+To enable the metrics correlation algorithm, update the following cluster setting:
+
+```
+PUT /_cluster/settings
+{
+  "persistent" : {
+    "plugins.ml_commons.enable_inhouse_python_model": true
+  }
+}
+```
+
+### Parameters
+
+To use the metrics correlation algorthim, include the following parameters.
+
+| Parameter | Type | Description | Default value |
+|---|---|---|---|
+metrics | Array | A list of metrics within the time series that can be correlated to anomalous behavior | N/A
+
+### Input
+
+The input to metrics correlation is an $M$ x $T$ array of metrics data, where M is the number of metrics and T is the length of each individual sequence of metric values. 
+
+When inputting metrics into the algorthim, assume the following:
+
+1. For each metric, the input sequence has the same length $T$.
+2. All input metrics should have the same corresponding set of timestamps.
+3. The total number of data points are $M$ * $T$ <= 10000.
+
+### Example: Simple metrics correlation
+
+The following example inputs number of metrics ($M$) as 3 and number of timesteps ($T$) as 128.
```

Comment:
Should "timesteps" be "timestamps"?

Reply:
Timesteps, the metrics act kind of like a ladder where each step is assigned a number in a time sequence.